scripts/lab-tui.py

```
@@ -474,19 +536,33 @@ def _run_upscale(self):
+                )
+                return
+            tfvars.write_text(updated_content)
+        except OSError as e:
             self.call_from_thread(log.write_line, f"[red]FAIL Could not update tfvars: {e}[/red]")
             return
 
-        rc, out, err = run_cmd(f"cd {TERRAFORM_DIR} && terraform apply -auto-approve")
+        rc, out, err = run_cmd(
+            f"terraform -chdir={shlex.quote(str(TERRAFORM_DIR))} apply -auto-approve"
+        )
         for line in out.splitlines(): self.call_from_thread(log.write_line, line)
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)